hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitInfosComponent.js

```diff
@@ -210,77 +223,82 @@ const Actions = (
     ];
 };
 
-const OrgUnitCreationDetails = ({ orgUnit, formatMessage, classes }) => {
+const Row = ({ label, value }) => {
+    const classes = useStyles();
+    return (
+        <TableRow>
+            <TableCell className={classes.leftCell}>{label}</TableCell>
+            <TableCell>{value}</TableCell>
+        </TableRow>
+    );
+};
+
+const OrgUnitCreationDetails = ({ orgUnit, formatMessage }) => {
+    const latitude = `${formatMessage(MESSAGES.latitude)}: ${
+        orgUnit.latitude
+    },`;
+    const longitude = `${formatMessage(MESSAGES.longitude)}: ${
+        orgUnit.longitude
+    },`;
+
     const latitudeLongitude =
-        orgUnit.latitude && orgUnit.longitude
-            ? `${formatMessage(MESSAGES.latitude)}: ${
-                  orgUnit.latitude
-              },  ${formatMessage(MESSAGES.longitude)}: ${orgUnit.longitude}`
-            : false;
+        orgUnit.latitude && orgUnit.longitude ? latitude + longitude : false;
+    const orgUnitCreatedAt = moment.unix(orgUnit.created_at).format('LTS');
+    const orgUnitUpdatedAt = moment.unix(orgUnit.updated_at).format('LTS');
```

Comment:
We could open a ticket to ask the backend to send the date in a string format. It sometimes send a unix timestamp, sometimes a string. In my opinion it's easier and less error prone for us to handle the string format

Reply:
Again I just notice it now, it was like this for a while already

Reply:
@hakifran https://bluesquare.atlassian.net/browse/IA-1802?atlOrigin=eyJpIjoiYmRmNDc2NDM4YWQwNDhkYTg2NDMxOTcwZjE4NmM0OGMiLCJwIjoiaiJ9